app/assets/stylesheets/application.css.scss

```diff
@@ -84,20 +84,33 @@ $grid-float-breakpoint:   320px;
   margin-top: 0;
 }
 
-.page-header--has-feed {
-  padding-right: 30px;
-  .feed {
-    float: right;
-    margin-top: -2.6em;
-    margin-right: -30px;
-  }
-}
-
 .feed-icon {
   width: 24px;
   height: 24px;
 }
 
+.subscription-options a {
+  float: right;
+  @media (min-width: $screen-sm-min) {
+    margin-top: 26px;
+  }
+}
+.btn.btn-email {
+  padding: 1px 3px 1px 5px;
+  margin-right: 5px;
+}
+.btn.btn-email:after {
+  content: ' ';
```

Comment:
Properties should be ordered background, background-size, content, display, height, margin-bottom, margin-left, position, width<br>Prefer double-quoted strings